roscala/src/main/scala/coop/rchain/rosette/prim/rblstring.scala

```diff
@@ -0,0 +1,70 @@
+package coop.rchain.rosette.prim
+
+import coop.rchain.rosette.{Ctxt, Fixnum, Ob, RblBool, RblString}
+import coop.rchain.rosette.macros.{checkArgumentMismatch, checkTypeMismatch}
+import coop.rchain.rosette.prim.Prim._
+
+object rblstring {
+
+  /** Base RblString Binary operation routine.
+    * It checks parameter types and returns a PrimError if the first parameter is not
+    * an RblString, RblBool(false) if the second parameter is not an RblString, or the
+    * result of the comparison as an RblBool if both parameters are an RblString.
+    */
+  case class strBinOp(name: String, f: (String, String) => Boolean) extends Prim {
+    override val minArgs: Int = 2
+    override val maxArgs: Int = 2
+
+    @checkArgumentMismatch
+    override def fn(ctxt: Ctxt): Either[PrimError, RblBool] = {
+      val elem = ctxt.argvec.elem
+
+      checkRblStringFirst(elem(0)).flatMap { s1 =>
+        checkRblStringSecond(elem(1)) match {
+          case Right(s2) => Right(RblBool(f(s1.value, s2.value)))
+          case Left(res) => Right(res)
```

Comment:
`case Left(_) => Right(RblBool(false))` simplifies it a little more I think.

Reply:
I think I like my way better, but have it your way.

Done.